xbmc/addons/Skin.cpp

```diff
@@ -369,6 +369,9 @@ const INFO::CSkinVariableString* CSkinInfo::CreateSkinVariable(const std::string
 
 void CSkinInfo::OnPreInstall()
 {
+  bool skinLoaded = g_SkinInfo != nullptr;
```

Comment:
Not sure what this supposed to do (?) g_SkinInfo will never be null except during startup, before the first skin is loaded